ts_benchmark/utils/__init__.py

```diff
@@ -4,5 +4,5 @@
 @Auth ： Allen Qiu
 @File ：__init__.py.py
 @IDE ：PyCharm
-@作用:
+@Purpose:
 """
```

Comment:
better remove this header, to be consistent with other files

Reply:
It has been removed.